### DISCUSSION TITLE
新手求问，使用Actions 无法打Ubuntu包

### DISCUSSION THREAD

**Discussion by ghost** [tw93/Pake#452] [Q&A]
提示 ：No files were found with the provided path: node_modules/pake-cli/output/*. No artifacts will be uploaded.
![Screenshot from 2023-05-30 04-12-16](https://github.com/tw93/Pake/assets/56305158/1bddc4b8-d551-49cd-8100-b0d71ea90bca)
#430 有一个类似的讨论，但我不理解是什么意思？ [Icon, Optional]那一栏我什么都没填
测了一下macos可以打包

**Comment by Tlntin** [comment]
你可以看看上一行的日志， build with pake ，大概率是命名问题。

**Reply by ghost** [reply]
已解决，非常感谢，我名字里有数字，导致一直警告

**Reply by ghost** [reply]
又试了几次，发现跟数字无关，名字里字母必须小写才行，大写就警告

**Reply by Tlntin** [reply]
> 又试了几次，发现跟数字无关，名字里字母必须小写才行，大写就警告

ubuntu包名都是小写，不信你用apt list看看你现在安装的包名。

**Reply by ghost** [reply]
好的，非常感谢
